src/actionTypes.js

```diff
@@ -656,6 +656,8 @@ export type Actions = {
   navigateToCreateStream: NavigateActionCreator,
   navigateToEditStream: NavigateToStreamActionCreator,
   navigateToNotifications: NavigateActionCreator,
+  navigateToHelp: NavigateAction,
+  navigateToRealmGate: NavigateAction,
```

Comment:
I guess `navigationToHelp` and `navigateToRealmGate` are action creators here. Actions and action creators can be tricky to distinguish at times. Here's how I like to remember it : An action is a plain javascript object whereas an action creator is a function which returns an action.
hence, the type here might change to NavigateActionCreator.

Reply:
Noted. Thanks!